manifests/example/upload-block-datavolume.yaml

```diff
@@ -0,0 +1,15 @@
+# This example assumes you are using a default storage class
```

Comment:
if you are using a block device its usually not the default storage class. then again it might be.